dz16.py

```diff
@@ -0,0 +1,103 @@
+"""life cycle of a bank card."""
+
+from datetime import datetime
+import random
+import string
+
+
+class BankCard:
+    """The class of bank card description."""
+
+    def __init__(self, card_num, name, surname, end_date, cvv, pin):
+        self.card_num = card_num
+        self.name = name
+        self.surname = surname
+        self.end_date = end_date
+        self.cvv = cvv
+        self.pin = pin
+        self.is_blocked = False
+
+    @classmethod
```

Comment:
Classmethod is method that operates class attributes ))
Here no self or cls usage - thus this is staticmethod
For class method you able to create method that will use class attributes - for example PIN_MAX_SIZE or BANK_ID etc